docs/flow/modin/experimental/batch.rst

```diff
@@ -0,0 +1,10 @@
+Batch Pipeline API description
```

Comment:
This section should be updated too - https://modin.readthedocs.io/en/stable/development/architecture.html#module-class-view.